patches/0003-Add-S3-legacy-bucket-to-resources.patch

```diff
@@ -3673,6 +3673,8 @@ index 0000000000..529368067d
 +
 +-> This functionality is for managing S3 in an AWS Partition. To manage [S3 on Outposts](https://docs.aws.amazon.com/AmazonS3/latest/dev/S3onOutposts.html), see the `aws_s3control_bucket` resource.
 +
++-> **NOTE:** Some feature of the pulumi-aws provider that worked with other cloud provider S3 integration might not work. Instead please have a look at the `aws.s3.BucketV2` resource.
```

Comment:
> Some feature of the pulumi-aws provider that worked with other cloud provider S3 integration might not work

Perhaps this can be clearer as "This resource might not work well if using an alternative s3-compatible provider. The aws.s3.BucketV2 should work better"